### PR TITLE
[new] add surface@1.01 with git auto-update

### PR DESCRIPTION
**Pull request by bennypowers**
Pull request for issue: #10287 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/mildrenben/surface
 * Official website (optional, not the repository): http://mildrenben.github.io/surface
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 161
   - Count of watchers: 17
   - Count of forks: 26

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Hello @bennypowers , I'll like to confirm if you are using git command line tool or GitHub gui, or some other git clients? The "merge" commit in this commit PR seem to be unnecessary.

**Comment by bennypowers** [comment]
github ui. You are correct that the merge commit is unnecessary

**Comment by PeterDaveHello** [comment]
@bennypowers no worry, let's just fix the error mentioned in CI, thanks!

File: ajax/libs/surface/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "surface",
+  "version": "1.0.1",
+  "description": "Material Design, CSS only framework.",
+  "main": "gulpfile.js",
```
Comment (pvnr0082t) [inline ajax/libs/surface/package.json]: @bennypowers We don't need `"main"`, `"scripts"`, `"bugs"`, `"devDependencies"` field. Let's remove it.

File: ajax/libs/surface/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "keywords": [
+    "material",
+    "material-design"
+  ],
+  "author": "Ben Mildren",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/mildrenben/surface/issues"
+  },
+  "homepage": "http://www.iamsurface.com",
```
Comment (pvnr0082t) [inline ajax/libs/surface/package.json]: I think we should use `"https://mildrenben.github.io/surface/"` as homepage.

File: ajax/libs/surface/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "surface",
+  "version": "1.0.1",
```
Comment (pvnr0082t) [inline ajax/libs/surface/package.json]: It should be `"1.01"`.

File: ajax/libs/surface/package.json (hunk 2 above)
Comment (bennypowers) [inline ajax/libs/surface/package.json]: done

File: ajax/libs/surface/package.json (hunk 1 above)
Comment (bennypowers) [inline ajax/libs/surface/package.json]: done

File: ajax/libs/surface/package.json (hunk 3 above)
Comment (bennypowers) [inline ajax/libs/surface/package.json]: done

**Comment by bennypowers** [comment]
thank you @pvnr0082t for recommending changes.

**Comment by bennypowers** [comment]
@pvnr0082t https://github.com/mildrenben/surface/issues/33

**Comment by PeterDaveHello** [comment]
@cdnjs/intern2 please help update PR title as we discussed many times.

File: ajax/libs/surface/package.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    "material",
+    "material-design"
+  ],
+  "author": "Ben Mildren",
+  "license": "MIT",
+  "homepage": "https://mildrenben.github.io/surface/",
+  "filename": "surface_styles.min.css",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com:mildrenben/surface.git",
```
Comment (kennynaoh) [inline ajax/libs/surface/package.json]: @bennypowers Please use `git://github.com/mildrenben/surface.git` as target. Thanks!

**Comment by pvnr0082t** [comment]
@PeterDaveHello Is that your `squash and rebase bot` mixed other commits?

**Comment by PeterDaveHello** [comment]
@pvnr0082t fixed, thanks.

**Comment by PeterDaveHello** [comment]
Hello @bennypowers, do you still want to finish this PR? Thanks.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by bennypowers** [comment]
Maybe @mildrenben can weigh in

**Comment by mildrenben** [comment]
Hi guys, author here,  late to the discussion sorry!

What am I required to do to get this working?

**Comment by sufuf3** [comment]
Hi @mildrenben , may I ask that in v1.01, the file `surface_styles.css` in which path?
Because I can't find it in v1.01.
And I think CDNJS can host `surface/docs/css/surface_styles.css`, but CDNJS host the library rely on release versions. So could you add a new [**git tag**](https://github.com/mildrenben/surface/tags)?
If in the newest release version, `surface_styles.css` can under `dist` or `css` folder which under root path, it will be a grateful help.

After you add a new git tag, I'll help @bennypowers promote this PR.

Thank you.

**Comment by sufuf3** [comment]
Hi, @mildrenben 
as this comment https://github.com/cdnjs/cdnjs/pull/10293#issuecomment-315057429 would you add a new git tag?
Thank you.

**Comment by sashberd** [comment]
@PeterDaveHello I think we cannot host this lib, no answer from author and also files are scss that still not hosted on cdnjs. Suggest to close

**Pull request closed without merging**